pkg/healthcheck/healthcheck_multicluster.go

```diff
@@ -451,15 +589,15 @@ func (hc *HealthChecker) checkRemoteClusterConnectivity() error {
 		}
 
 		if len(errors) > 0 {
-			return fmt.Errorf("Problematic clusters:\n\t%s", strings.Join(errors, "\n\t"))
+			return fmt.Errorf("Problematic clusters:\n\t%s", strings.Join(errors, "\n\t\t"))
```

Comment:
Will this print the first problematic cluster indented with one tab but all following clusters with two tabs?